data/actionsmap/yunohost.yml

```diff
@@ -535,6 +535,31 @@ app:
                     full: --file
                     help: Folder or tarball for upgrade
 
+        ### app_change_url()
+        change_url:
```

Comment:
Shouldn't we call this "changeurl" (without _) to be consistent with other existing actions such as checkurl, clearaccess, removelist, ... ?